app/workers/vendor_integration_stats_worker.rb

```diff
@@ -0,0 +1,110 @@
+require 'http'
+
+class VendorIntegrationStatsWorker
+  include Sidekiq::Worker
+
+  SLACK_CHANNELS = {
+    'tribal' => 'tribal_dfe_collaboration_',
+    'ellucian' => 'ellucian_dfe_collaboration',
+  }.freeze
+
+  def perform(vendor_name)
+    @vendor_name = vendor_name
+    @webhook_url = fetch_webhook_url
+
+    if @webhook_url.present?
+      message = SlackReport.new(vendor_name).generate
+      post_to_slack message
+    end
+  end
+
+private
+
+  def fetch_webhook_url
+    ENV.fetch "#{@vendor_name.upcase}_INTEGRATION_STATS_SLACK_URL"
+  end
+
+  def post_to_slack(slack_message)
+    slack_channel = '#twd_apply_test' # TODO: hardcoded for now
+
+    payload = {
+      username: 'Apply for teacher training',
+      channel: slack_channel,
+      text: slack_message,
+      mrkdwn: true,
+    }
+
+    response = HTTP.post(@webhook_url, body: payload.to_json)
+
+    unless response.status.success?
+      raise SlackMessageError, "Slack error: #{response.body}"
+    end
+  end
+
+  class SlackMessageError < StandardError; end
+
+  class SlackReport
```

Comment:
could we move this into it's own service?

Reply:
Leaving this as is for now, as nothing else uses it.